### PR TITLE
Added basic security configuration file

### PR DESCRIPTION
**Pull request by michal-jewczuk**
Just turned off spring security for all /api/* calls when we create and test the api features

### CODE REVIEW

**Comment by kingchen01** [comment]
Checked and accepted

**Comment by kingchen01** [comment]
test

File: api/src/main/java/com/fiszki/fiszkiproject/config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	@Override
+	protected void configure(HttpSecurity http) throws Exception {
+		http
+			.csrf().disable()
+			.exceptionHandling()
+			.and()
+			.authorizeRequests()
+			.antMatchers("/api/**").permitAll()
+			.and()
+			.formLogin().disable()
```
Comment (kingchen01) [inline api/src/main/java/com/fiszki/fiszkiproject/config/SecurityConfig.java]: test

**Comment by michal-jewczuk** [comment]
Ja też, ja też

**Pull request merged**